Add Dynamic Board Generation

EManGames opened the pull request:
-Add .gitignore

EManGames committed:
Add Dynamic Board Generation

-Add .gitignore

EManGames committed:
Bettter .gitignore

EManGames committed:
Update .gitignore

EManGames committed:
Update .gitignore

EManGames committed:
Update .gitignore

I don't know what I'm doing

EManGames committed:
Ignore Temp Files

https://stackoverflow.com/questions/43380622/github-desktop-is-committing-ignored-files

Most often the case is that if the files existed before they were added to the .gitignore file, they will not be ignored in any following commits or pushes.

My suggestions for you, since you don't want to use the command line, is cut the files/directories from your git directory, perform a commit, and then push. After the push finished, you can paste the offending files/directories back into the git repo and they should now be ignored.

Just in case you wanted to know, you can use the git bash git rm -r --cached some-directory and then perform a commit and push and you will have achieved the same result.

EManGames committed:
Remove Ignored Files

EManGames committed:
Add Dynamic Board Generation and Probuilder

- Makes a board when you hit play
- Install probuilder
- Included some temp X and O pieces for testing

CodingWithIanAlderson commented:
There was no .gitignore so by default I had 10000+ changed temp files

Bundled in this mess of commits is:

- A .gitignore so no one else needs to suffer https://github.com/kololueth/BattleTicTacToe/pull/1/commits/3edf7c4a463a36df69ef84ff5d0f1ee3d2d286e3
- Me cleaning up the mess https://github.com/kololueth/BattleTicTacToe/pull/1/commits/89f9e9b1c770a9a17322daf2ec46a31b8417bc79
![image](https://github.com/kololueth/BattleTicTacToe/assets/143231250/e1274995-a827-4ada-a35d-948865ace989)

- My actual commit of installing probuilder and dynamic board generation https://github.com/kololueth/BattleTicTacToe/pull/1/commits/b904229f258efb9e0048f5ddadf2f61b42a6d057

EManGames committed:
Final Commit

Probuilder Broke after I delete my temp files. Interested to see if they work for you

CodingWithIanAlderson commented:
My Above fixes broke my probuilder. I fixed it in https://github.com/kololueth/BattleTicTacToe/pull/1/commits/b904229f258efb9e0048f5ddadf2f61b42a6d057

kololueth commented:
Awesome, thanks Ian
